### ISSUE TITLE
Audio is not playing?

### ISSUE THREAD

**Issue by GopiK14** [zhuozhuo/ZHChat#61]
Hi Team,

We need to play audio file received from andriod, we get in 2 formats .mp3 or .aac. Both the audio format not playing in iOS with 'ZHCAudioMediaItem'. We tried to convert the same to 'm4a' and tried but no luck.

#1 - Confirm receive audio has data -  we renamed local file extension to m4a - it working fine - just confirm audio is not a problem.

#2 - Please check screenshot - its print audiodata bytes so audiodata is not nil.

![screen shot 2018-05-15 at 1 51 02 am](https://user-images.githubusercontent.com/20974304/40021448-79d10fae-57e2-11e8-960c-3a2257914b8b.png)

Any solution, Please suggest some other solutions.

**Comment by deankai** [comment]
convert data to m4a not fileName.aac. If u don't know what's wrong with ur code. u should check audio guide. aac is sort format not fileName.

**Comment by GopiK14** [comment]
Thanks! I renamed the received file to .m4a once downloaded but it not playing. If possible, can you share a audio conversion code would be really helpful.

**Comment by GopiK14** [comment]
Any solution for this?

**Comment by deankai** [comment]
did u play audio from file , not data? When u get data ,then save it to file ,then get data from file. Make sure that u get right file name. And it can really be played , like demo.

**Comment by GopiK14** [comment]
Yes, I write the data to local file with .m4a format but not player. Is it support mp3 format right or we need to consider other formats?
